ecs-cli/modules/cli/local/converter/converter.go

```diff
@@ -82,13 +81,16 @@ const (
 	endpointsMetadataV3URI = "http://169.254.170.2/v3"
 )
 
+// composeVersion is the minimum Compose file version supporting task definition fields.
+const composeVersion = "3.2"
+
 // SecretLabelPrefix is the prefix of Docker label keys
 // whose value is an ARN of a secret to expose to the container.
 // See https://github.com/aws/amazon-ecs-cli/issues/797
 const SecretLabelPrefix = "ecs-local.secret"
 
-// ConvertToDockerCompose creates the payload from an ECS Task Definition to be written as a docker compose file
-func ConvertToDockerCompose(taskDefinition *ecs.TaskDefinition, metadata *LocalCreateMetadata) ([]byte, error) {
+// ConvertToCompose creates the payload from an ECS Task Definition to be written as a docker compose file
```

Comment:
I think we could be more clear in the comment here and `MarshalComposeConfig` -- this no longer creates the payload but rather a compose config (so perhaps also rename this function to `ConvertToComposeConfig` ?)

Reply:
Done, renamed to `ConvertToComposeConfig` and updated the comment to:
```go
// ConvertToComposeConfig translates an ECS Task Definition to a the Docker Compose config and returns it.
```